premium/usage.go

```diff
@@ -240,6 +251,31 @@ func NewUsageClient(meta plugin.Meta, ops ...UsageClientOptions) (UsageClient, e
 	return u, nil
 }
 
+func (u *BatchUpdater) setupAWSMarketplace() error {
+	cfg, err := awsConfig.LoadDefaultConfig(context.TODO())
+	if err != nil {
+		return fmt.Errorf("failed to load AWS config: %w", err)
+	}
+
+	u.awsMarketplaceClient = marketplacemetering.NewFromConfig(cfg)
+	u.teamName = "AWS_MARKETPLACE"
+	// This needs to be larger than normal, because we can only send a single usage record per second (from each compute node)
+	u.batchLimit = 1000000000
+	u.backgroundUpdater()
+	return nil
+}
+
+func isAWSMarketplace() bool {
+	return os.Getenv("CQ_AWS_MARKETPLACE_CONTAINER") == "true"
```

Comment:
Where these envs are being set? If it's something user settable (and not a published image of some sort) strconv.ParseBool would've been better, but I'm guessing it's an image?